app/scripts/lib/rpc-method-middleware/handlers/wallet-createSession/handler.ts

```diff
@@ -11,7 +11,7 @@ import {
   NormalizedScopesObject,
   getSupportedScopeObjects,
   Caip25CaveatValue,
-  setPermittedAccounts,
+  setNonSCACaipAccountIdsInCaip25CaveatValue
```

Comment:
This change is unrelated but required due to breaking changes on the `@metamask/chain-agnostic-permission@0.7.0` package that went into this. Diff should go away after https://github.com/MetaMask/metamask-extension/pull/32152 is merged.